inc/saihostintf.h

```diff
@@ -698,6 +612,181 @@ typedef sai_status_t(*sai_get_hostif_attribute_fn)(
         _Inout_ sai_attribute_t *attr_list);
 
 /**
+* @brief Attribute data for SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE
+*/
+typedef enum _sai_hostif_table_entry_type_t
+{
+    /** Port-based Host Interface entry Type */
+    SAI_HOSTIF_TABLE_ENTRY_TYPE_PORT,
+
+    /** LAG based Host Interface entry Type */
+    SAI_HOSTIF_TABLE_ENTRY_TYPE_LAG,
+
+    /** Vlan based Host Interface entry Type */
+    SAI_HOSTIF_TABLE_ENTRY_TYPE_VLAN,
+
+    /** Wildcard Interface entry Type */
+    SAI_HOSTIF_TABLE_ENTRY_TYPE_TRAP_ID,
+
+    /** Wildcard Interface, wildcard trap id */
+    SAI_HOSTIF_TABLE_ENTRY_TYPE_WILDCARD
+
+} sai_hostif_table_entry_type_t;
+
+/**
+* @brief Attribute data for SAI_HOSTIF_TABLE_ENTRY_ATTR_CHANNEL_TYPE
+*/
+typedef enum _sai_hostif_table_entry_channel_type_t
+{
+    /** receive packets via callback */
+    SAI_HOSTIF_TABLE_ENTRY_CHANNEL_TYPE_CB,
+
+    /** receive packets via file descriptor */
+    SAI_HOSTIF_TABLE_ENTRY_CHANNEL_TYPE_FD,
+
+    /** receive packets via Linux netdev type port */
+    SAI_HOSTIF_TABLE_ENTRY_CHANNEL_TYPE_NETDEV_PHYSICAL_PORT,
+
+    /** receive packets via Linux netdev logical port (LAG or port) */
+    SAI_HOSTIF_TABLE_ENTRY_CHANNEL_TYPE_NETDEV_LOGICAL_PORT,
```

Comment:
The purpose of physical port type is not clear when the type logical port itself can cover both lag and port. We are not exposing physical port object anywhere else in SAI. An example will help.